docs/quickstart/README.md

```diff
@@ -395,16 +296,22 @@ This is all done specifically to pass the Tornjak config file as an argument to
 
 </details>
 
-<details><summary><b>[Click] For the Tornjak-backend + frontend sidecar implementation (EXPERIMENTAL)</b></summary>
+<details><summary><b> 🔴 [Click] For the Tornjak-backend + frontend sidecar implementation (if you are unsure, choose this) </b></summary>
```

Comment:
maybe instead of `(if you are unsure, choose this)`, we can say (this is our default deployment) ?